blog/2023/2023-07-24-setting-goals-for-your-open-source-contributions.md

```diff
@@ -24,7 +24,7 @@ You’re more likely to succeed in what you do if you if you set goals and you c
 
 5. **Prioritize and Time Management**. Prioritize your goals and create a schedule that divides time for contributions, learning, and community engagement. Remember, the goal of #100DaysOfOSS is to help you grow. You don’t want to burn out. So be realistic about the time that you have to devote to open source projects. And remember, it’s ok and good to schedule days off as part of your schedule.
 
-6. **Set Accountability Measures**. Establish ways to track your progress, such as using our [journal template](https://github.com/open-sauced/100-days-of-oss-template), adding [a highlight to your OpenSauced profile](https://insights.opensauced.pizza/feed), and/or using the #100DaysOfOSS hashtag on social media. You can also share your goals in the [OpenSauced Discord](https://discord.gg/opensauced) for added support.
+6. **Set Accountability Measures**. Establish ways to track your progress, such as using our [journal template](https://github.com/open-sauced/100-days-of-oss-template), adding [a highlight to your OpenSauced profile](https://insights.opensauced.pizza/feed), and/or using the #100DaysOfOSS hashtag on social media. You can also share your goals in the [OpenSauced forum](https://github.com/orgs/open-sauced/discussions) for added support.
```

Comment:
Do we want to say "OpenSauced forum" or "OpenSauced Community" as in line 19?

Reply:
That link in the triage is fine. We want to send them to the welcome message. Everything else should use the general link unless a specific discussion is noted.

Also, I think it's fine to use Community and forum interchangeably. Ultimately the community _is_ a forum, so it's not a huge deal.

Reply:
I don't want to change any content in this PR unless it's the switch from discord to forum. I'm going to be updating the #100Days in the next month or so, so we can revisit it then.